docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -1,19 +1,36 @@
 ---
 title: Debug Office Add-ins on iPad and Mac
 description: ''
-ms.date: 01/25/2018
+ms.date: 03/21/2018
 ---
 
 # Debug Office Add-ins on iPad and Mac
 
 You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on the iPad or Mac. Because add-ins are developed using HTML and Javascript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on an iPad or Mac. 
 
-## Debugging with Vorlon.JS 
+## Debugging with Safari Web Inspector on a Mac
 
-Vorlon.JS is a debugger for web pages, similar to the F12 tools. It is designed to work remotely and it enables you to debug web pages across different devices. For more information, see the [Vorlon website](http://www.vorlonjs.com).  
+You can debug an Office add-in using Safari Web Inspector. 
 
+To enable debugging Office add-ins on Mac, you must have Mac OS High Sierra AND Mac Office Version: 16.9.1 (Build 18012504) or later. If you don't have an Office Mac build, you can get one by joining the [Office 365 Developer program](https://aka.ms/o365devprogram).
 
-### Install and set up up Vorlon.JS on a Mac or iPad 
+To start, open a terminal and set the OfficeWebAddinDeveloperExtras property for the relevant Office application as follows:
+
+- `defaults write com.microsoft.Word OfficeWebAddinDeveloperExtras -bool true`
+- `defaults write com.microsoft.Excel OfficeWebAddinDeveloperExtras -bool true`
+- `defaults write com.microsoft.Powerpoint OfficeWebAddinDeveloperExtras -bool true`
+- `defaults write com.microsoft.Outlook OfficeWebAddinDeveloperExtras -bool true`
+
+Then, open the Office application, and insert your add-in. Right click the add-in and you should see an **Inspect Element** option in the context menu.  Select that and it will pop the Inspector, where you can set breakpoints and debug your add-in.
```

Comment:
Since you say "right-click the add-in" -- safe to assume that these instructions only apply to task pane add-ins?  What about content add-ins, etc.?  May want to call this out somewhere in this section.

Reply:
For clarity, this sentence should use voice and style guidelines, I.e.:
Next, open the Office application and insert your add-in. Right-click the add-in, and select the **Inspect Element** option in the context menu. Run the Inspector to set breakpoints and debug your add-in.